### PR TITLE
Hops

### PR DESCRIPTION
**Pull request by dfquaresma**
This PR implements the logic to log the hops. It adds a new column representing that. 
The value on each line of a hop column will be values of service time that the respective request received at some server that it has passed for. Those values (service time by server) are separated by spaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ as sufix. The bullets below explains what each column represent.
 * **status**: It gives the http status of a request. 200 means successfully request, 503 means failed requests.  
 * **done**: It gives the request state. True if the request was processed and False otherwise.
```
Comment (danielfireman) [inline README.md]: Remove this and the line bellow. These are not part of the output anymore.

File: log.py [hunk 2]
```diff
@@ -24,11 +24,11 @@ def log_debbug(requests, debbug_path, debbug_name, mode):
     data = []
     if mode == "w":
         data.append(["id", "timestamp", "created_time", "latency",
```
Comment (danielfireman) [inline log.py]: Why are we still needing the done column? Status already tells us this information.

File: models/loadbalancer.py [hunk 3]
```diff
@@ -48,11 +46,13 @@ def shed_request(self, request):
...
             self.forward(request)
 
+        yield self.env.timeout(self.sleep)  # Aim at make this function a generator function.
+
     def request_succeeded(self, request):
         request.finished_at(self.env.now)
         self.succeeded_requests += 1
         self.requests.append(request)
-        yield self.env.timeout(self.sleep)
+        yield self.env.timeout(self.sleep)  # Aim at make this function a generator function.
```
Comment (danielfireman) [inline models/loadbalancer.py]: Didn't get the comment. The request_succeeded function is already a generator.

File: README.md (hunk 1 above)
Comment (dfquaresma) [reply README.md]: They are used when simulator generates a log results with more detailed information.

File: log.py (hunk 2 above)
Comment (danielfireman) [reply log.py]: What if both logs are wrong? What you're saying is that you have a log message to confirm that another log message is correct? To me, it looks like a receipt for confusion.

If you would like to make sure that something is correct, test it. Having two log messages saying the same thing does look like the way to go.

Comment (dfquaresma) [reply log.py]: I see. Your argument makes sense to me. Going to remove it in next commit.

**Pull request merged**